Gulpfile.js

```diff
@@ -239,10 +238,6 @@ gulp.task("build-no-bundle", () => buildBabel());
 gulp.task(
   "watch",
   gulp.series("build-no-bundle", function watch() {
-    gulpWatch(
-      defaultSourcesGlob,
-      { debounceDelay: 200 },
-      gulp.task("build-no-bundle")
-    );
+    gulp.watch(defaultSourcesGlob, gulp.task("build-no-bundle"));
```

Comment:
The `gulp.watch` equivalent of `{ debouceDelay: 200 }` is { delay: 200 }`. According to https://gulpjs.com/docs/en/api/watch#options `200` is the default `delay` value, so we remove option argument here.